config/locales/server.en.yml

```diff
@@ -2401,6 +2401,7 @@ en:
     remove_muted_tags_from_latest: "Don't show topics tagged only with muted tags in the latest topic list."
     force_lowercase_tags: "Force all new tags to be entirely lowercase."
     create_post_for_category_and_tag_changes: "Create a small action post when a topic's category or tags change"
+    automatically_clean_tags: "Automatically clean up unused tags"
```

Comment:
This is not very descriptive; I think something like "Automatically delete tags that are not being used on any topics or private messages on a daily basis."